pkg/pillar/kubeapi/vitoapiserver.go

```diff
@@ -313,6 +313,89 @@ func RolloutDiskToPVC(ctx context.Context, log *base.LogObject, exists bool,
 	return nil
 }
 
+// GetPVFromPVC : Returns volume name (PV) from the PVC name
+func GetPVFromPVC(pvcName string, log *base.LogObject) (string, error) {
+	// Get the Kubernetes clientset
+	clientset, err := GetClientSet()
+	if err != nil {
+		err = fmt.Errorf("failed to get clientset: %v", err)
```

Comment:
A new error it's being created here, why you just don't propagate the original error (from the `GetClientSet()` call)?